scripts/init.d-xapissl

```diff
@@ -24,8 +24,6 @@ PEMFILE="@ETCDIR@/xapi-ssl.pem"
 SSLPIDFILE="/var/run/xapissl.pid"
```

Comment:
You can also drop the whole comment section at the top, the include of lsb init functions, or just in general we do not need this to be a shell script at all, we could move it all inside ocaml, all it needs to do is generate a config file, and call a script to potentially generate a certificate.

With the new stunnel soon to be merged there will also be a systemd service `stunnel@` that we could use, perhaps open a ticket to do this work independent of what you are currently doing.

Reply:
i will make a ticket for this